stl/inc/xmemory

```diff
@@ -1241,8 +1241,10 @@ using _Iterator_base = _Iterator_base12;
 
 
 // ALIAS TEMPLATE _Container_proxy_ptr AND SUPPORTING MACHINERY
-struct _Leave_proxy_unbound {}; // tag to indicate that a proxy is being allocated before it is safe to bind to a
-                                // _Container_base12
+struct _Leave_proxy_unbound {
+    explicit _Leave_proxy_unbound() = default;
+}; // tag to indicate that a proxy is being allocated before it is safe to bind to a
+   // _Container_base12

```

Comment:
This comment can now be unwrapped to a single line. (clang-format wraps excessively long comments, but won't unwrap in situations like this.)